src/win.rs

```diff
@@ -38,6 +37,7 @@ struct WinIface {
     name: String,
     /// The human-readable name of the interface
     description: String,
+    index: u32,
```

Comment:
`usize` would be a more appropriate type for indices

Reply:
done